### PR TITLE
(DEV-3623) Added Pre-selection Prop

### PR DESCRIPTION
**Pull request by redreceipt**
Fixes [issue-number]

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

**Comment by IsaacHardy** [comment]
@redreceipt Looks like there's a linting issue.

File: src/@ui/Meta/__tests__/__snapshots__/index.js.snap [hunk 1]
```diff
@@ -75,7 +75,7 @@ Object {
       "name": "og:image",
     },
     Object {
-      "content": "http://localhost:3000",
+      "content": "https://my.newspring.cc",
```
Comment (IsaacHardy) [inline src/@ui/Meta/__tests__/__snapshots__/index.js.snap]: Let's keep this as localhost:3000

File: src/@ui/forms/ContributionForm/index.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
 import React, { Component } from 'react';
 import { View, Platform } from 'react-native';
 import PropTypes from 'prop-types';
-import { compose, withProps, branch, renderComponent, setPropTypes, defaultProps } from 'recompose';
+import {
```
Comment (IsaacHardy) [inline src/@ui/forms/ContributionForm/index.js]: Are there any changes in this file besides prettier updates? We should revert changes like these or not commit them in the future.

File: src/@ui/forms/ContributionForm/FundInput.js [hunk 3]
```diff
@@ -73,6 +73,7 @@ const pickerWrapperStyle = {
 
 const FundInput = ({
   isFirst = false,
+  preselection = {},
   funds = [],
   value = {},
```
Comment (IsaacHardy) [inline src/@ui/forms/ContributionForm/FundInput.js]: Is this used?

File: src/@ui/Meta/__tests__/__snapshots__/index.js.snap (hunk 1 above)
Comment (redreceipt) [reply src/@ui/Meta/__tests__/__snapshots__/index.js.snap]: Why? Seems like putting anything to do with local dev environments in the repo isn't right. Is that why the texts are failing?

File: src/@ui/forms/ContributionForm/FundInput.js (hunk 3 above)
Comment (redreceipt) [reply src/@ui/forms/ContributionForm/FundInput.js]: Yes it's used all throughout the file

File: src/@ui/forms/ContributionForm/index.js (hunk 2 above)
Comment (redreceipt) [reply src/@ui/forms/ContributionForm/index.js]: Yes L#204 and L#210 are the biggest changes but most of it is prettier

File: src/@ui/Meta/__tests__/__snapshots__/index.js.snap (hunk 1 above)
Comment (IsaacHardy) [reply src/@ui/Meta/__tests__/__snapshots__/index.js.snap]: This is a snapshot. So it doesn't matter what is in these, but constantly changing them can get tedious.

Comment (IsaacHardy) [reply src/@ui/Meta/__tests__/__snapshots__/index.js.snap]: You have a linting issue. That is why they are failing.

Comment (richarddubay) [reply src/@ui/Meta/__tests__/__snapshots__/index.js.snap]: The hard thing about this snapshot is that it is pulling based off an environment variable, right? So whatever Travis has as the variable is what we should match against. Unless the variable in Travis needs to change.

**Comment by redreceipt** [comment]
@IsaacHardy not sure why those snapshots are failing. they aren't failing locally.

**Comment by IsaacHardy** [comment]
@redreceipt Your snapshots aren't failing. Your linting is failing.

File: src/@ui/Meta/__tests__/__snapshots__/index.js.snap (hunk 1 above)
Comment (IsaacHardy) [reply src/@ui/Meta/__tests__/__snapshots__/index.js.snap]: That's right. The is being pulled from the `.env` file.

**Comment by IsaacHardy** [comment]
It looks good on iOS and web, but I'm having difficulties with my Android env.

**Comment by IsaacHardy** [comment]
Looks good!

**Comment by IsaacHardy** [comment]
I'd like @richarddubay or @cdanwards to review before we merge.

**Pull request merged**